sql/api/src/main/scala/org/apache/spark/sql/types/Decimal.scala

```diff
@@ -681,8 +681,6 @@ object Decimal {
     override def toLong(x: Decimal): Long = x.toLong
     override def fromInt(x: Int): Decimal = new Decimal().set(x)
     override def compare(x: Decimal, y: Decimal): Int = x.compare(y)
-    // Added from Scala 2.13; don't override to work in 2.12
-    // TODO revisit once Scala 2.12 support is dropped
     def parseString(str: String): Option[Decimal] = Try(Decimal(str)).toOption
```

Comment:
I think this is meant to be `override` in Scala 2.13; does that work?

Reply:
in scala2.12, Numberic not has this method, it ok, refer to `scala.math.Numeric.BigDecimalIsConflicted` in scala2.13

Reply:
Right, so shouldn't it be a `override` now?

Reply:
yes,I will.